main.js

```diff
@@ -1274,19 +1274,16 @@ addStealExtension(function (loader) {
 		}
 	};
 
-if(typeof System !== "undefined") {
-	addEnv(System);
-}
-
-function addEnv(loader){
-	// Add the extension to _extensions so that it can be cloned.
-	loader._extensions.push(addEnv);
+// Steal Env Extension
+// adds some special environment functions to the loader
+addStealExtension(function (loader) {
```

Comment:
@matthewp i think you forgot to make a release after merging https://github.com/stealjs/steal/pull/1023 the last steal release is 7 days ago. the PR is 6 days ago.